PhysicsTools/PatAlgos/python/slimming/metFilterPaths_cff.py

```diff
@@ -27,7 +29,7 @@
 
```

Comment:
this file appears behind 75X/76X now:
to be updated by https://github.com/cms-sw/cmssw/pull/10558

I can guess that it will have a priority